src/help/applyOptions.js

```diff
@@ -0,0 +1,32 @@
+export default applyOptions;
+
+function applyOptions(optionsBlueprint, options) {
+	var returnOptions = {};
+	for (var name in optionsBlueprint) {
+		if (optionsBlueprint.hasOwnProperty(name)) {
+			if (typeof options[name] !== 'undefined') {
+				returnOptions[name] = convertOption(options[name], optionsBlueprint[name].type);
+			}
+			else {
+				returnOptions[name] = optionsBlueprint[name].default;
+			}
+		}
+	}
+	return returnOptions;
+}
+
+// Convert an option to a specified type
+function convertOption(data, type) {
+	if (type === 'string') {
```

Comment:
For example `convertOption(10, 'number')` causes `return parseInt(data, 10);` anyway. I suggest check type of data before and avoid any unnecessary conversion. Or I missed something :)